io.vikunja.Vikunja.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.vikunja.Vikunja
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=x11
```

Comment:
`share=ipc` is necessary.

Reply:
and I suspect `device=dri` is needed too.

Reply:
> and I suspect `device=dri` is needed too.

What does it do?